keywords/TestServerAndroid.py

```diff
@@ -18,12 +18,11 @@ def __init__(self, version_build, host, port, community_enabled=None, debug_mode
         self.platform = platform
 
         if self.platform == "android":
+            self.download_source = "couchbase-lite-android"
             if community_enabled:
                 apk_name_prefix = "CBLTestServer-Android-{}-community".format(self.version_build)
```

Comment:
CBL do not support the community version right ?

Reply:
so far we don't build/test cc version, but this code supports cc from day 1, I'd prefer to keep it